jobs/delorean/2.0/ews/amq-online/discovery.yaml

```diff
@@ -15,8 +15,8 @@
       - ci-trigger:
           override-topic: Consumer.rh-jenkins-ci-plugin.8cfe81de-2e51-4b77-96a9-191a1a559fb1.VirtualTopic.eng.brew.build.complete
           message-checks:
-          - field: '$.info.package_id'
-            value: '74050'
+            - field: '$.info.package_id'
+              value: '73640'
```

Comment:
Is this the switch the correct AMQ brew build?

Reply:
Yep exactly, the one that was mentioned here: https://issues.redhat.com/browse/DEL-62?focusedCommentId=13976209&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13976209